### ISSUE TITLE
Reason for using imagenet mean and std for normalization

### ISSUE THREAD

**Issue by Radhikadua123** [jrzech/reproduce-chexnet#12]
In model.py, it is mentioned that imagenet mean, std are used for normalization. However, the mean and std of NIH dataset is much larger than that of imagenet. Can you please share the reason for using imagenet mean and std for normalization? Maybe I am missing something.

**Comment by jrzech** [comment]
While counterintuitive, this is quite commonly done when starting with a network pretrained to ImageNet and was replicated from the original ChexNet paper: "Before inputting the images into the network, we downscale the images to 224×224 and normalize based on the mean and standard deviation of images in the ImageNet training set"
